aws/eks.go

```diff
@@ -0,0 +1,95 @@
+package aws
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eks"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// getAllEksClusters returns a list of strings of EKS Cluster Names that uniquely identify each cluster.
+func getAllEksClusters(awsSession *session.Session, excludeAfter time.Time) ([]*string, error) {
```

Comment:
Don't you need to call this EKS code from somewhere? I think there's a struct of some sort you need to add and then some extra API calls in the main CLI app...

Reply:
Yup... I can't believe I missed that...

Reply:
I was so tied up in the tests and it was getting late that I got a bit too trigger happy I guess.